### PR TITLE
Fix demos

### PR DESCRIPTION
**Pull request by ppamorim**
This tiny PR fixes the demos to support the current version of meilisearch-swift in the master branch.

### CODE REVIEW

**Comment by ppamorim** [comment]
bors try

File: Demos/VaporDemo/Sources/App/routes.swift [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+
+}
+
 func routes(_ app: Application) throws {
 
-    let client = try! MeiliSearch(Config(hostURL: "http://localhost:7700"))
+    let client = try! MeiliSearch(Config.default(apiKey: "masterKey"))
 
-    // 127.0.0.1:8080/index?uid=Movies
+    // 127.0.0.1:8080/index?uid=books_test
```
Comment (curquiza) [inline Demos/VaporDemo/Sources/App/routes.swift:28]:
```suggestion
    // 127.0.0.1:7700/indexes/books_test
```

File: Demos/PerfectDemo/Sources/PerfectTemplate/main.swift [hunk 2]
```diff
@@ -71,6 +92,7 @@ func index(request: HTTPRequest, response: HTTPResponse) {
 
 }
 
+// 127.0.0.1:8181/search?query=botman
```
Comment (curquiza) [inline Demos/PerfectDemo/Sources/PerfectTemplate/main.swift:95]:
```suggestion
// 127.0.0.1:7700/search?query=botman
```

**Review by curquiza — CHANGES_REQUESTED** [review]
👍

File: Demos/VaporDemo/Sources/App/routes.swift (hunk 1 above)
Comment (ppamorim) [reply Demos/VaporDemo/Sources/App/routes.swift:28]: The port should be `8080` for Vapor test. We are testing the vapor example, not meilisearch.

File: Demos/PerfectDemo/Sources/PerfectTemplate/main.swift (hunk 2 above)
Comment (ppamorim) [reply Demos/PerfectDemo/Sources/PerfectTemplate/main.swift:95]: Same as before but for port `8181`.

**Comment by bidoubiwa** [comment]
Hey @ppamorim, 
Could you add in the readme how to try the demo's ?

**Comment by curquiza** [comment]
Agreed with @bidoubiwa. Updating the [current section](https://github.com/meilisearch/meilisearch-swift#demo) in the README. Or at least providing a README in each sub-folders containing a demo.

File: README.md [hunk 3]
```diff
@@ -109,10 +109,20 @@ This package only guarantees the compatibility with the [version v0.15.0 of Meil
...
+
 ### Perfect
 
 Please check the Perfect Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/PerfectDemo).
 
+To test the Perfect demo you will need to go to the directory `Demos/PerfectDemo`, you can either:
+
+ - Open the SwiftPM project in Xcode and press run, or
+ - Run `swift build` or `swift run` in the terminal.
+
```
Comment (bidoubiwa) [inline README.md:125]: I'm not sure about writing the same text two times. What do you think of the following?

Also about step this `Open the SwiftPM project in Xcode and press run, or`. Do you need to open XCode with `/demos/VaporDemo`?
And about this step ` Run swift build or swift run in the terminal.` do you need to run that command inside the Demo folder ?

```
## Demo

To try out a demo you will need to go to its directory in `Demos/`. After which you can either: 

 - Open the SwiftPM project in Xcode and press run, or
 - Run `swift build` or `swift run` in the terminal.

### Vapor

Please check the Vapor Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/VaporDemo).

### Perfect 

Please check the Perfect Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/PerfectDemo).

```

**Review by curquiza — COMMENTED** [review]
Thanks for adding this quick explanation!!
Maybe I don't know how to do it, but I did not find any way to create an index and add documents with the Perfect Demo ( I haven't tried the Vapor one yet). If it's indeed an issue, we should open an issue and fix in a different PR 🙂

File: README.md [hunk 4]
```diff
@@ -109,10 +109,14 @@ This package only guarantees the compatibility with the [version v0.15.0 of Meil
 
 Please check the Vapor Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/VaporDemo).
 
+To test the Vapor demo, go to the directory `Demos/VaporDemo` and open the SwiftPM project in Xcode and press run or run `swift build` or `swift run` in the terminal.
```
[the remark this answers is not in the document]
Comment (curquiza) [reply README.md]: Sorry, I did not see the answer of charlotte before posting 😉 making this as resolved

File: README.md [hunk 5]
```diff
@@ -109,10 +109,14 @@ This package only guarantees the compatibility with the [version v0.15.0 of Meil
...
+
 ### Perfect
 
 Please check the Perfect Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/PerfectDemo).
 
+To test the Vapor demo, go to the directory `Demos/PerfectDemo` and open the SwiftPM project in Xcode and press run or run `swift build` or `swift run` in the terminal.
```
[the remark this answers is not in the document]
Comment (curquiza) [reply README.md]: same

File: README.md (hunk 3 above)
Comment (curquiza) [reply README.md:125]: I like the non-repetitive way 👍

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (bidoubiwa) [reply README.md]: We could do a mix of both. I like the way you made this more clear

[the remark this answers is not in the document]
Comment (curquiza) [reply README.md]: Let's discuss on your change suggestion. I let you take what you like in my suggestion 🙂

**Comment by ppamorim** [comment]
bors try

File: README.md [hunk 6]
```diff
@@ -105,11 +105,16 @@ This package only guarantees the compatibility with the [version v0.15.0 of Meil
...
+
+ - Open the SwiftPM project in Xcode and press run, or
+ - Run `swift build` or `swift run` in the terminal.
+
 ### Vapor
 
 Please check the Vapor Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/VaporDemo).
 
-### Perfect
+### Perfect
```
Comment (curquiza) [inline README.md:117]:
```suggestion
### Perfect
```

**Review by curquiza — CHANGES_REQUESTED** [review]

**Review by curquiza — APPROVED** [review]

**Comment by curquiza** [comment]
bors merge

**Pull request merged**